src/__tests__/core/account.test.ts

```diff
@@ -45,3 +44,46 @@ test('account: select account balance', async () => {
   expect(result.errors).toBeUndefined();
   expect(result.data.account.balance).toBeGreaterThan(0);
 });
+
+test('account: select account balance with unit conversion', async () => {
+  const units =  {'wei': 1,
+                  'kwei': 1000, 'babbage': 1000, 'femtoether': 1000,
+                  'mwei': 1000000, 'lovelace': 1000000, 'picoether': 1000000,
+                  'gwei': 1000000000, 'shannon': 1000000000, 'nanoether': 1000000000, 'nano': 1000000000,
+                  'microether': 1000000000000, 'micro': 1000000000000, 'szabo': 1000000000000,
+                  'milliether': 1000000000000000, 'milli': 1000000000000000, 'finney': 1000000000000000,
+                  'ether': 1000000000000000000,
+                  'kether': 1000000000000000000000, 'grand': 1000000000000000000000,
+                  'mether': 1000000000000000000000000,
+                  'gether': 1000000000000000000000000000,
+                  'tether': 1000000000000000000000000000000};
+
+  let wei;
+  for (const unit of Object.keys(units)) {
+    const query = `
+    {
+      account(address: "0x0000000000000000000000000000000000000000") {
+        balance(unit:` + unit + `)
```

Comment:
Try using string interpolation here, given that you're using template literals (backticks) ;-)